jsonnet/components/cluster-monitoring-operator.libsonnet

```diff
@@ -222,6 +222,26 @@ function(params) {
         resources: ['events'],
         verbs: ['create', 'patch', 'update'],
       },
+      {
+        apiGroups: [''],
+        resources: ['pods'],
+        verbs: ['list', 'delete'],
+      },
+      {
+        apiGroups: [''],
+        resources: ['persistentvolumeclaims'],
+        verbs: ['get', 'list', 'update', 'delete'],
+      },
+      {
+        apiGroups: [''],
+        resources: ['nodes'],
+        verbs: ['get', 'list', 'update', 'patch'],
+      },
+      {
+        apiGroups: [''],
+        resources: ['persistentvolumes'],
+        verbs: ['get'],
+      },
```

Comment:
good point

Reply:
I moved `pods` and `pvcs` related rules to two different Roles, one for the `openshift-monitoring` namespace and the other for `openshift-user-workload-monitoring`. I generated them from jsonnet since the existing ClusterRole is already generated that way. I also added the relevant ClusterRoleBindings to `0000_50_cluster-monitoring-operator_03-role-binding.yaml`, but these changes are raw yaml to be consistent with the existing content of the file.